08.ls_object/file_list.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class FileList
+  def initialize(params)
+    @params = params
+  end
+
+  def display_lists
+    current_files = Dir.glob('*', @params[:a] ? File::FNM_DOTMATCH : 0)
```

Comment:
```suggestion
    hoge = @params[:a] ? File::FNM_DOTMATCH : 0
    current_files = Dir.glob('*', hoge)
```

ちょっと読みづらいので一度変数に入れた方がいいと思います。